### PR TITLE
Update ui_lizmap_popup.py

### PR DESCRIPTION
**Pull request by samtux**
Update  QtGui. Not found QApplication, QGridLayout, QGroupBox, QTextEdit, QDialogButtonBox  from qgis.PyQt.QtGUI

### CODE REVIEW

File: ui_lizmap_popup.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
 #
 # WARNING! All changes made in this file will be lost!
```
Comment (Gustry) [inline ui_lizmap_popup.py]: As mentioned here, all changes in this file will be lost after a regeneration using `pyuic5`. You need to fix in the UI file first.

And I proposed a few months ago to remove the use the compiled UI file in https://github.com/3liz/lizmap-plugin/pull/119 but I have no answer from the 3Liz team.

Comment (samtux) [reply ui_lizmap_popup.py]: Thank you @Gustry  for the answer.

**Comment by rldhont** [comment]
#119 has been merged.
Can you update your commit and proposed to merge on master and not master_3 ?
Thanks

**Comment by samtux** [comment]
Proposed to merge on master

**Comment by Gustry** [comment]
Due to #119, the file you edited doesn't exist anymore.
It needs to be fixed in ui_lizmap_popup.py if you still have the bug. Can you test the `master` branch?

**Comment by Gustry** [comment]
I think this PR can be closed because the file doesn't exist anymore.
We use the UI file only now.

**Comment by rldhont** [comment]
Thanks for reminding

**Pull request closed without merging**